### ISSUE TITLE
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Magento\Catalog\Block\Product\ImageFactory::getLabel() must be of the type string

### ISSUE THREAD

**Issue by dimasch** [DivanteLtd/magento2-vsbridge-indexer#7]
On `Magento 2.3.0` version have reindex error:

With magento `sample data` 

```
bin/magento indexer:reindex
```

```
Design Config Grid index has been rebuilt successfully in 00:00:00
Catalog Rule Product index has been rebuilt successfully in 00:00:01
Vsbridge Category Indexer index has been rebuilt successfully in 00:00:01
Vsbridge Attributes Indexer index has been rebuilt successfully in 00:00:03
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Magento\Catalog\Block\Product\ImageFactory::getLabel() must be of the type string, null given, called in /Users/dima/sites/m230/vendor/magento/module-catalog/Block/Product/ImageFactory.php on line 154 and defined in /Users/dima/sites/m230/vendor/magento/module-catalog/Block/Product/ImageFactory.php:106
Stack trace:
#0 /Users/dima/sites/m230/vendor/magento/module-catalog/Block/Product/ImageFactory.php(154): Magento\Catalog\Block\Product\ImageFactory->getLabel(Object(Magento\Catalog\Model\Product\Interceptor), NULL)
#1 /Users/dima/sites/m230/vendor/magento/module-catalog/Block/Product/ImageBuilder.php(135): Magento\Catalog\Block\Product\ImageFactory->create(Object(Magento\Catalog\Model\Product\Interceptor), 'new_products_co...', Array)
#2 /Users/dima/sites/m230/vendor/magento/module-catalog/Block/Product/AbstractProduct.php(513): Magento\Catalog\Block\Product\ImageBuilder->create(Object(Magento\Catalog\Model\Product\Interceptor), 'new_products_co...', Array)
#3 /Users/di in /Users/dima/sites/m230/vendor/magento/module-catalog/Block/Product/ImageFactory.php on line 106
    <div class="block widget block-products-list grid">
                <div class="block-content">
            <!-- new_products_content_widget_grid-->            <div class="products-grid grid">
                <ol class="product-items widget-product-grid">
                                                                <li class="product-item">                        <div class="product-item-info">
                            <a href="http://m230.test/selene-yoga-hoodie.html" class="product-item-photo">
                                

```

**Comment by dimasch** [comment]
Related to sample data widget `<!-- new_products_content_widget_grid-->`

**Comment by afirlejczyk** [comment]
Hi,
Error appears only for `vsbridge_cms_block_indexer` indexer.
fix is available in master branch b286b89be8d30e9b5a7577efa929d91a2c88ad1a

**Comment by dimasch** [comment]
Thanks, Agata @afirlejczyk
